modules/lobby/src/main/LobbySocket.scala

```diff
@@ -67,6 +68,7 @@ final class LobbySocket(
         hookSubscriberSris.clear()
 
       case ReloadTimelines(users) => send(Out.tellLobbyUsers(users, makeMessage("reload_timeline")))
+      case UpdateFeedRev(rev)     => send(Out.tellLobby(makeMessage("update_feed", rev)))
```

Comment:
we absolutely don't need to live-update the homepage when a new feed entry is added. It's perfectly fine to serve it on the next page load.